src/transaction/types/output/consts.rs

```diff
@@ -0,0 +1,25 @@
+use fuel_types::bytes::WORD_SIZE;
+use fuel_types::{Address, AssetId, Bytes32, ContractId};
+
+pub(super) const OUTPUT_CCV_TO_OFFSET: usize = WORD_SIZE; // Identifier
```

Comment:
What does CCV stand for?

Reply:
Coin, change and variable. Instead of writing all three and making a super big name, and considering its only used inside the module, I reduced it a bit